get-substrate.sh

```diff
@@ -67,19 +67,19 @@ else
 	rustup default stable
 fi
 
+rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
```suggestion
rustup update nightly
rustup target add wasm32-unknown-unknown --toolchain nightly
```

Reply:
Needs to get nightly if it is not installed

Reply:
I tested it on a "new" computer and `rustup update nightly` was enough to install nightly if it wasnt there already.

Reply:
Yes you are right, that is why I suggested it here. You missed the `rustup update nightly` command. 
Doesn't matter anymore, I committed that fix to the script.

Reply:
oh, nice, thank you :)